jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/sender/PushNotificationSenderEndpoint.java

```diff
@@ -100,9 +101,17 @@
      * @HTTP 401 (Unauthorized) The request requires authentication.
      * @HTTP 404 (Not Found) The requested PushApplication resource does not exist.
      * @RequestHeader aerogear-sender The header to identify the used client. If the header is not present, the standard "user-agent" header is used.
+     *
+     * @param message   message to send
+     * @return          successful string {@code Job submitted}
+     *
+     * @statuscode 202 Indicates the Job has been accepted and is being process by the AeroGear UnifiedPush Server
+     * @statuscode 401 The request requires authentication
+     * @statuscode 404 The requested PushApplication resource does not exist
      */
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
+    @ReturnType("java.lang.String")
     public Response send(final UnifiedPushMessage message, @Context HttpServletRequest request) {
```

Comment:
> Hm... I think that in this case we should split UnifiedPushMessage to UnifiedPushMessage and something like UnifiedPushMessageExtended extends UnifiedPushMessage

@idelpivnitskiy how about having an inherited class inside of the push/rest module -> `InternalUnifiedPushMEssage extends UnifiedPushMessage`

@sebastienblanc I think that extra class would be also handy on the analtycs tracking stuff, right?